internal/manifest/v1beta1/manifest_controller_test.go

```diff
@@ -1,151 +0,0 @@
-package v1beta1_test
-
-import (
-	"errors"
-	"os"
-	"path/filepath"
-	"strings"
-
-	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
-	declarative "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
-	"github.com/kyma-project/lifecycle-manager/pkg/ocmextensions"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/uuid"
-	"k8s.io/apimachinery/pkg/util/yaml"
-
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-)
-
-var (
-	ErrManifestStateMisMatch = errors.New("ManifestState mismatch")
-	ErrAuthSecretNotFound    = errors.New("auth secret error not found in manifest")
-	ErrNotInErrorState       = errors.New("manifest not found in error state")
-)
-
-var _ = Describe(
-	"Given manifest with OCI specs", func() {
-		mainOciTempDir := "main-dir"
-		installName := filepath.Join(mainOciTempDir, "installs")
-		It(
-			"setup OCI", func() {
-				PushToRemoteOCIRegistry(installName)
-			},
-		)
-		BeforeEach(
-			func() {
-				Expect(os.RemoveAll(filepath.Join(os.TempDir(), mainOciTempDir))).To(Succeed())
-			},
-		)
-		DescribeTable(
-			"Test OCI specs",
-			func(
-				givenCondition func(manifest *v1beta2.Manifest) error,
-				expectManifestState func(manifestName string) error,
-			) {
-				manifest := NewTestManifest("oci")
-				Eventually(givenCondition, standardTimeout, standardInterval).
-					WithArguments(manifest).Should(Succeed())
-				Eventually(expectManifestState, standardTimeout, standardInterval).
-					WithArguments(manifest.GetName()).Should(Succeed())
-				Eventually(deleteManifestAndVerify(manifest), standardTimeout, standardInterval).Should(Succeed())
-			},
-			Entry(
-				"When Manifest CR contains a valid install OCI image specification, "+
-					"expect state in ready",
-				withValidInstallImageSpec(installName, false),
-				expectManifestStateIn(declarative.StateReady),
-			),
-			Entry(
-				"When Manifest CR contains a valid install OCI image specification and enabled deploy resource, "+
-					"expect state in ready",
-				withValidInstallImageSpec(installName, true),
-				expectManifestStateIn(declarative.StateReady),
-			),
-			Entry(
-				"When Manifest CR contains an invalid install OCI image specification, "+
-					"expect state in error",
-				withInvalidInstallImageSpec(false),
-				expectManifestStateIn(declarative.StateError),
-			),
-		)
-	},
-)
-
-var _ = Describe(
-	"Test multiple Manifest CRs with same parent and OCI spec", func() {
-		mainOciTempDir := "multiple"
-		installName := filepath.Join(mainOciTempDir, "crs")
-		It(
-			"setup remote oci Registry",
-			func() {
-				PushToRemoteOCIRegistry(installName)
-			},
-		)
-		BeforeEach(
-			func() {
-				Expect(os.RemoveAll(filepath.Join(os.TempDir(), mainOciTempDir))).To(Succeed())
-			},
-		)
-		It(
```

Comment:
This test case is not present in the PR. Is it redundant/obsolete?

Reply:
glad you mention it! this test to me is meaningless, because:
1. it just tests installing a manifest layer with the same content twice, but not verifying anything. `withValidInstallImageSpec` only creates manifest CR, not checking for its state.
2. I believe the test trying to verify the situation that same module deployed into different kyma, so it can have multiple manifest CR with same OCI image (belongs to that module), but this situation can only happens LM in control-plane mode, it's not a setup can be easily configured in this integration test. And in single cluster mode, there is no such situation that multiple manifest CR have same OCI image, because each manifest CR belongs to a dedicated module.